### PR TITLE
[FIX] Fix Claim Modal translations keys

### PR DESCRIPTION
**Pull request by flavioislima**
Fix Claim Rewards dialog showing i18n keys instead of the translations.

image

---

Use the following Checklist if you have changed something on the Backend or Frontend:

- [x] Tested the feature and it's working on a current and clean install.
- [ ] Tested the main App features and they are still working on a current and clean install. (Login, Install, Play, Uninstall, Move games, etc.)
- [ ] Created / Updated Tests (If necessary)
- [ ] Created / Updated documentation (If necessary)

### CODE REVIEW

File: src/frontend/components/UI/QuestsViewer/index.tsx [hunk 1]
```diff
@@ -34,6 +34,13 @@ export function QuestsViewer({ projectId: appName }: QuestsViewerProps) {
     )
   }
 
+  /**
+   t("quest.claimWarning.body", "<bold>IMPORTANT:</bold> Please ensure that you are allocating enough gas on the {{networkName}} network for the transaction to be successfully confirmed <bold>within 7 days.</bold>")
+    t("quest.claimWarning.body2", "Otherwise, the Quest Reward <bold>will expire and will no longer be claimable.</bold>")
+    t('quest.claimWarning.cancel', 'Cancel')
+    t('quest.claimWarning.confirm', 'Confirm')
+   */
```
Comment (BrettCleary) [inline src/frontend/components/UI/QuestsViewer/index.tsx:42]: can we add a comment here explaining why we have this comment and asking future devs to not delete it?

**Review by BrettCleary — APPROVED** [review]

File: src/frontend/components/UI/QuestsViewer/index.tsx (hunk 1 above)
Comment (flavioislima) [reply src/frontend/components/UI/QuestsViewer/index.tsx:42]: Sure, great idea!

Comment (flavioislima) [reply src/frontend/components/UI/QuestsViewer/index.tsx]: done

**Review by eliobricenov — APPROVED** [review]

**Review by nyghtstalker — APPROVED** [review]
I am seeing the updated text on the confirmation modal. 

image

**Pull request merged**